cot/templates/admin/model.html

```diff
@@ -34,6 +34,40 @@ <h2>{{ model.name() }}</h2>
 
     <footer>
         Displaying {{ objects.len() }} {{ model.name() }}{{ objects.len()|pluralize }}.
+
+        <div class="pagination">
+            <select id="page-size-selector">
+                {% for option in [10, 20, 30, 40] %}
+                    <option value="{{ option }}" {% if option == page_size %}selected{% endif %}>{{ option }}</option>
+                {% endfor %}
+            </select>
+
+            {% if page > 1 %}
+                <a href="?page={{ page - 1 }}&page_size={{ page_size }}" class="btn">Previous</a>
+            {% else %}
+                <button class="btn disabled">Previous</button>
+            {% endif %}
+
+             <span>Page {{page}} of {{total_pages}}</span>
+
+            {% if page < total_pages %}
+                <a href="?page={{ page + 1 }}&page_size={{ page_size }}" class="btn">Next</a>
```

Comment:
Same as above.

```suggestion
                <a href="?page={{ page + 1 }}&page_size={{ page_size }}" class="btn secondary">Next</a>
```

Reply:
hey @m4tx Thanks for the comments! I've fixed all the issues, plz let me know what else or its good to go!